plugin/cscope_maps.vim

```diff
@@ -41,12 +41,40 @@ if has("cscope")
     " if you want the reverse search order.
     set csto=0
 
-    " add any cscope database in current directory
-    if filereadable("cscope.out")
-        cs add cscope.out
-    " else add the database pointed to by environment variable
-    elseif $CSCOPE_DB != ""
+    " cscope seems to not index files in symlinked directories.
+    " However, one can build the cscope.files (list of files for cscope to index)
+    " via a command like:
+    " `find -L [your searched folder] -name [your searched pattern]
+    "   -exec realpath {} \; >> cscope.files`
+    "
+    " Note that it is prudent to use absolute file paths, hence the use of realpath.
+    "
+    " Now the trouble is getting vim to try to find a cscope.out file
+    " going recursively up through directories, but going back up through
+    " them via any symlinks you took to get to your CWD in the shell.
+    " The following codes solves this issue.
+
+    if $CSCOPE_DB != ""
```

Comment:
What about this case? Is it left unhanded w.r.t. symlinks?

Reply:
If `CSCOPE_DB` is defined, then I assume that one only wants to use that cscope file. We could make different assumptions.

Reply:
Ok, never mind this one.